src/ui/screens/initialize/ViewInitialize.qml

```diff
@@ -99,6 +99,16 @@ Item {
             objectName: "swipeView"
             anchors.fill: parent
             currentIndex: 0
+            property string telemetryScreenId: currentIndex === 0 ? "sign_up" : `sign_up_${currentIndex + 1}`
+
+            onTelemetryScreenIdChanged: {
+                Glean.impression.signupScreenView.record({
+                    screen: telemetryScreenId,
+                    action: "impression",
+                });
+
+                previousTelemetryScreenId = telemetryScreenId
```

Comment:
where is `previousTelemetryScreenId` declared?

Reply:
Ha, that is deelopment leftover. Thanks for catching it, Matt.